Thirdparty/DBoW2/DBoW2/TemplatedVocabulary.h

```diff
@@ -1451,6 +1463,78 @@ void TemplatedVocabulary<TDescriptor,F>::saveToTextFile(const std::string &filen
 // --------------------------------------------------------------------------
 
 template<class TDescriptor, class F>
+bool TemplatedVocabulary<TDescriptor,F>::loadFromBinaryFile(const std::string &filename) {
```

Comment:
It might be good to document where the code come from.

Reply:
It is from https://github.com/raulmur/ORB_SLAM2/pull/21. But it hasn't been accepted and now the branch has conflicts. So I change the code manually.

Reply:
OK. Please add comment of this info.

Reply:
@pancx , please add comments with "Take and rebase code from ....".